src/main/kotlin/app/revanced/util/AbstractIntegrationsPatch.kt

```diff
@@ -0,0 +1,39 @@
+package app.revanced.util
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstruction
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultError
+import app.revanced.patcher.patch.PatchResultSuccess
+import org.jf.dexlib2.iface.Method
+
+private val MISSING_INTEGRATIONS = PatchResultError(
```

Comment:
Since this is only used in `AbstractIntegrationsPatch`, what is the reason you put it in the class here instead of in a private companion object?

Reply:
It's what I normally do and what I see others do as well. Want me to change it?

Reply:
Also, adding it to a (private) companion object creates a new Companion object. I believe if you do this, it'll instead be in `AbstractIntegrationsPatchKt.class` as a static variable. Although, this doesn't really that much.